### PR TITLE
Add .gitignore

### PR DESCRIPTION
**Pull request by razo7**

### CODE REVIEW

File: .gitigonre [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Created by https://www.toptal.com/developers/gitignore/api/web
+# Edit at https://www.toptal.com/developers/gitignore?templates=web
+
+### Web ###
+*.asp
+*.cer
+*.csr
+*.css
+*.htm
+*.html
```
Comment (bennypowers) [inline .gitigonre]: Most of these are reasonable source files, why ignore them and not node_modules?

**Review by bennypowers — DISMISSED** [review]

File: .gitigonre (hunk 1 above)
Comment (razo7) [reply .gitigonre]: That's correct, they are reasonable source files.
I have used [gitignore.io](https://www.toptal.com/developers/gitignore) to generate a default `.gitignore`  file.
Should it only include `node_modules/`?

**Comment by TomerFi** [comment]
Maybe we should use a more global `gitignore` that covers multiple languages.

image

**Review by TomerFi — APPROVED** [review]

**Comment by TomerFi** [comment]
I think this looks good.
Can we merge this?

**Pull request merged**